src/trigger/LivetimeCounter.hpp

```diff
@@ -0,0 +1,52 @@
+#ifndef TRIGGER_SRC_TRIGGER_LIVETIMECOUNTER_HPP_
+#define TRIGGER_SRC_TRIGGER_LIVETIMECOUNTER_HPP_
+
```

Comment:
Some docstring comment would be helpful!

Reply:
Done in a0b2e44